src/Resources/templates/htmlTableOutput.phtml

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SensioLabs DeprecationDetector</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
I've played a bit with @slde-incal's proposal and this is the result:

### When some violations are detected

![deprecator-errors](https://cloud.githubusercontent.com/assets/73419/10673731/a69c081c-78f7-11e5-90a8-e1eded998ea8.png)

### When no violations are detected

![deprecator-no-errors](https://cloud.githubusercontent.com/assets/73419/10673735/ad08a066-78f7-11e5-8085-905acaed1f00.png)